ferrocene/library/libc/src/lib.rs

```diff
@@ -23,7 +23,13 @@
 )]
 #![cfg_attr(libc_deny_warnings, deny(warnings))]
 // Attributes needed when building as part of the standard library
-#![cfg_attr(feature = "rustc-dep-of-std", feature(link_cfg, no_core))]
+// ferrocene addition
```

Comment:
the only Ferrocene addition is on line 30

Reply:
Moved the comment to correct line.